### PR TITLE
Update Migaloo Testnet - Narwhal-2

### PR DESCRIPTION
**Pull request by 0xbobloblaw**

### CODE REVIEW

**Comment by reversesigh** [comment]
A few things.

1.) please use an appropriate title for your PR

2.) since this is a new chain, you should change the status of 1 to killed, archive it, and create a new testnet for 2.

File: testnets/migalootestnet/chain.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "../chain.schema.json",
-  "chain_name": "migalootestnet",
+  "chain_name": "migaloo-testnet",
```
Comment (JeremyParish69) [inline testnets/migalootestnet/chain.json:3]: Cannot use '-' in the chain_name here. Keep as 'migalootestnet'

**Comment by JeremyParish69** [comment]
@reversesigh sometimes I let smaller testnets update with a new chain-id without requiring them to archive. Mainnet I'd say is 1000% required, though. Seems like the submitter may have abandoned this PR.

**Comment by 0xbobloblaw** [comment]
Made fixes @JeremyParish69 @reversesigh

**Pull request merged**